.github/CODEOWNERS

```diff
@@ -30,10 +30,15 @@
 /Templates/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
 /Tools/EventLogTool/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
 
-# SIG-Core Asset Pipeline ownership area
-/AutomatedTesting/Gem/PythonTests/assetpipeline/ap_fixtures/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
-/AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
-/Code/Tools/AssetProcessor/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+# SIG-Content Asset Pipeline ownership area
```

Comment:
Not sure of the best way to organize this file. But it maybe useful to separate by SIG and thus move this block to the end.

Reply:
Sure, the end makes this file reverse alphabetical order by sig. Sig testing is at the top, then sig core, and now sig content at the bottom.